api/api/utils/image_proxy/__init__.py

```diff
@@ -1,5 +1,7 @@
+import json
 import logging
 from dataclasses import dataclass
+from itertools import count as itertools_count
```

Comment:
nit: This could just as well be `import itertools` and then referenced as `itertools.count`, right? Is there a benefit to pulling it out explicitly and still using a verbose name?

Reply:
Good point, I'll rectify.